src/Modules/CalcDefence.lua

```diff
@@ -854,46 +857,24 @@ function calcs.defence(env, actor)
 				},
 			}
 		end
-		local enemyCritChance = env.configInput["enemyCritChance"] or 5
-		local enemyCritDamage = env.configInput["enemyCritDamage"] or 30
+		local enemyCritChance = env.configInput["enemyCritChance"] or env.configPlaceholder["enemyCritChance"] or 0
+		local enemyCritDamage = env.configInput["enemyCritDamage"] or env.configPlaceholder["enemyCritDamage"] or 0
```

Comment:
Can this ever be 0 now?

Reply:
because of order of initialization it will be nil for a first pass, but is instantly rebuilt so, no the value will never be zero but it errors due to nil value on initialization

Reply:
because of order of initialization it will be nil for a first pass, but is instantly rebuilt so, no the value will never be zero but it errors due to nil value on initialization